controller.js

```diff
@@ -24,5 +24,50 @@ module.exports = {
     }).then(countries => {
       res(countries);
     });
+  },
+
+
+/**
+  * creates api log from req body by taking paramters from it
+  * @function setApi
+  * @param {String} userid The userid (primary key) to get a user.
+  * @return {json} status and an object in json format
+  */
+  setApi:function(req,res){
+    var param = req.body;
+    if(Object.keys(param).length == 0 && param.constructor === Object)
+    {
+/* This is just a dummy JSON object which will be passed as
+ * a parameter when the request body is empty
+ * It's just for naive testing(POSTMAN) and will be deleted in near future when not needed
+ */
+      param =  {
+      caller: "Shreyansh",
+      bloodgroup: "A+",
+      stdcode: "022",
+      pincode:"400706",
+      apicode:"w1hbr3br4",
+      donorfound:1,
+      donormobile:'8767903336',
+      ipaddress:'127.0.0.1',
+      remarks:"good"
+    };
```

Comment:
Can we remove this stub?

Reply:
Yes, I can.I had only kept this stub in case ,if we were to test the function using POSTMAN, it would return a proper result.The function takes the parameters from request body and in the unit test, three different inputs are passed anyway.

Reply:
Updated it.